README.md

```diff
@@ -1,2 +1,202 @@
-# todo_list_testing
-Code Review with todo-list app
+<div align="center">
+
+  <h3><b>Minimalist Todo Application</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [TODO App] <a name="about-project"></a>
+
+Minimalist To-do Application is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **Html 5**
+- **CSS 3**
+- **Bootstrap 5**
+- **Webpack 5**
+- **Rxjs**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Todo**
+- **Todo List**
+- **Clear Completed Todos**
+- **Refresh App**
+- **Clear Completed tasks**
+- **Mark task as complete**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Here is the link to the live demo version - [Minimalist Todo App](https://minimalist-todo-list.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com/Stephen-Adom/minimalist-todo-list.git
+```
+
+-
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+-
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+```
+
+-
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```

Comment:
- [ ] I suggest it will have been better to add more clear details on how to run tests and deploy the project to make more simple and easy to understand for anyone accessing it and it also makes your ReadMe file look professional and for more detailed you can check out the better way by clicking on this [link](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/).

Reply:
@TaiwoEnoch Thank you for pointing this issue👍. I will work on it